sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=dynamic-extensions-for-alfresco
```

Comment:
Remove this unused properties file

Reply:
pushed with commit `a44d8003c2bd3240662875cd16550ff1d8cb0038`